numpydoc/tests/test_docscrape.py

```diff
@@ -331,13 +331,15 @@ def _strip_blank_lines(s):
 
 
 def line_by_line_compare(a, b):
+    empty_description = '..'
+    rgx = re.compile(r"^\s+" + re.escape(empty_description) + "$")
```

Comment:
rgx is a terrible name for something that matches an empty description. But I'm also not sure why it's important to disregard these.

Reply:
Renamed to `empty_description_rgx`.